### DISCUSSION TITLE
How many ribs does the average human have at birth? 🦴

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#940] [Galaxy Trivia] [answered]
**Bonus:** On which part of the body is a tabi worn?

**Comment by peterbaccaro** [comment, marked as answer]
At birth, the average human typically has a total of **24 ribs**. There are 12 pairs of ribs, with each rib attaching to the thoracic vertebrae at the back and curving around to the front, connecting to the sternum or breastbone. The first seven pairs of ribs are known as true ribs, as they directly attach to the sternum via their own costal cartilage. The next three pairs, known as false ribs, connect to the sternum indirectly through the cartilage of the rib above. The last two pairs of ribs, called floating ribs, do not attach to the sternum at all and are free-floating in the lower part of the ribcage.

Tabi is a traditional type of Japanese footwear that is **worn on the feet**. It is a split-toe sock or shoe that has a separate section for the big toe, while the other toes are grouped together. Tabi is typically worn with traditional Japanese clothing, such as kimono or yukata. They are often used in conjunction with geta (wooden sandals) or zori (flat sandals with fabric straps). Tabi helps to separate the big toe, which aids in balance and stability when wearing the traditional footwear. They are commonly worn by both men and women on various occasions, such as festivals, ceremonies, or when practicing martial arts like Aikido or Kendo.
